### ISSUE TITLE
No f1, precision and recall scores during training

### ISSUE THREAD

**Issue by adap7** [tomaarsen/SpanMarkerNER#63]
I am currently training a model:
encoder_id = "bert-base-cased"
model = SpanMarkerModel.from_pretrained(
    encoder_id,
    labels=labels,
    model_max_length=512,
    entity_max_length=512,
    marker_max_length=256
)
During the training process, only the overall accuracy is displayed, but no additional metrics are provided (f1, recall and precision are 0), Training Loss and Validation Loss seem to be improving, but overall accuracy is also constant throughout training. When evaluating on the test set the same problem appears, f1, precision and recall are 0, overall accuracy is the same as for training. As I am new to this, I am unsure if this is expected behavior. The size of my training set is 329 for training and 219 for testing.

Request:
Could you please provide guidance on whether this is the expected behavior and, if not, how to obtain additional metrics during the training process? Thank you for your assistance.
